Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -1511,3 +1511,14 @@ theorem Pred.rec_top (p : α → Prop) (htop : p ⊤) (hpred : ∀ a, p a → p
 #align pred.rec_top Pred.rec_top
 
 end OrderTop
+
+lemma SuccOrder.forall_ne_bot_iff
```

Comment:
Can you add the `PredOrder` version too, or is it not trivially true via the OrderDual?